etl/steps/data/garden/wb/2023-07-10/education.countries.json

```diff
@@ -0,0 +1,250 @@
+  "Sierra Leone": "Sierra Leone",
+  "Singapore": "Singapore",
+  "Sint Maarten (Dutch part)": "Sint Maarten (Dutch part)",
+  "Slovak Republic": "Slovakia",
+  "Slovenia": "Slovenia",
+  "Solomon Islands": "Solomon Islands",
+  "Somalia": "Somalia",
+  "South Africa": "South Africa",
+  "South Sudan": "South Sudan",
+  "Spain": "Spain",
+  "Sri Lanka": "Sri Lanka",
+  "St. Kitts and Nevis": "Saint Kitts and Nevis",
+  "St. Lucia": "Saint Lucia",
+  "St. Martin (French part)": "Saint Martin (French part)",
+  "St. Vincent and the Grenadines": "Saint Vincent and the Grenadines",
+  "Sudan": "Sudan",
+  "Suriname": "Suriname",
+  "Sweden": "Sweden",
+  "Switzerland": "Switzerland",
+  "Syrian Arab Republic": "Syria",
+  "Tajikistan": "Tajikistan",
+  "Tanzania": "Tanzania",
+  "Thailand": "Thailand",
+  "Timor-Leste": "East Timor",
+  "Togo": "Togo",
+  "Tonga": "Tonga",
+  "Trinidad and Tobago": "Trinidad and Tobago",
+  "Tunisia": "Tunisia",
+  "Turkey": "Turkey",
+  "Turkmenistan": "Turkmenistan",
+  "Turks and Caicos Islands": "Turks and Caicos Islands",
+  "Tuvalu": "Tuvalu",
+  "Uganda": "Uganda",
+  "Ukraine": "Ukraine",
+  "United Arab Emirates": "United Arab Emirates",
+  "United Kingdom": "United Kingdom",
+  "United States": "United States",
+  "Uruguay": "Uruguay",
+  "Uzbekistan": "Uzbekistan",
+  "Vanuatu": "Vanuatu",
+  "Venezuela, RB": "Venezuela",
+  "Vietnam": "Vietnam",
+  "Virgin Islands (U.S.)": "United States Virgin Islands",
+  "West Bank and Gaza": "Palestine",
+  "World": "World",
+  "Yemen, Rep.": "Yemen",
+  "Zambia": "Zambia",
+  "Zimbabwe": "Zimbabwe",
+  "East Asia & Pacific": "East Asia & Pacific",
+  "Korea, Dem. People\u2019s Rep.": "North Korea",
+  "Middle East & North Africa (excluding high income)": "Middle East & North Africa (excluding high income)",
+  "Middle income": "Middle income",
+  "Low & middle income": "Low & middle income",
+  "East Asia & Pacific (excluding high income)": "East Asia & Pacific (excluding high income)",
+  "High income": "High income",
+  "Lower middle income": "Lower middle income",
+  "Latin America & Caribbean": "Latin America & Caribbean",
+  "Least developed countries: UN classification": "Least developed countries: UN classification",
+  "Upper middle income": "Upper middle income",
+  "Heavily indebted poor countries (HIPC)": "Heavily indebted poor countries (HIPC)",
+  "South Asia": "South Asia",
+  "Sub-Saharan Africa": "Sub-Saharan Africa",
+  "Arab World": "Arab World",
+  "Middle East & North Africa": "Middle East & North Africa",
+  "Low income": "Low income",
+  "Europe & Central Asia (excluding high income)": "Europe & Central Asia (excluding high income)",
+  "Europe & Central Asia": "Europe & Central Asia",
+  "Sub-Saharan Africa (excluding high income)": "Sub-Saharan Africa (excluding high income)",
+  "Latin America & Caribbean (excluding high income)": "Latin America & Caribbean (excluding high income)",
+  "European Union": "European Union",
```

Comment:
Please check that they refer to EU27 (most likely), and, if so, rename this to "European Union (27)".